modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,71 @@
+= Portal
+
+`Unit ability`
+
+Some races, such as Necrons and Eldar, can open portals directly onto the battlefield.
+
+---
+
+[NOTE]
+====
+This ability lets you keep detachments off-table at the start of the game, similar to Reserves.
+You can then deploy them later in the game -- when they arrive, you disembark them into the play area and have them through a unit -- or objective -- that has this ability.
+====
+
+== Portal Reserves
+
+Where your army includes any units -- or objectives -- that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that can use a {portal} off-table.
```

Comment:
What about when the detachment is already in play and you embark through one portal and immediately disembark through another, as mentioned elsewhere in this topic? Can we then use March orders as well, or must we still use only Advance or Assault orders?

Reply:
Ugh... I'm working locally, and tried adding this comment in VS Code — but it's posted here in the wrong context (I guess it matched the line number, which has entirely different content to my local version).
I'll delete this and another similar comment I made, and keep them in the local code for now.